src/components/DownloadLineageButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { IconButton } from '@mui/material';
+import CloudDownloadIcon from '@mui/icons-material/CloudDownload';
+import { toPng } from 'html-to-image';
+
+function downloadImage(dataUrl: string) {
+  const a = document.createElement('a');
+  a.setAttribute('download', 'lineage.png');
+  a.setAttribute('href', dataUrl);
+  a.click();
+}
+
+function DownloadButton() {
+  const download = () => {
+    toPng(document.querySelector('.react-flow') as HTMLElement, {
+      filter: (node) => {
+        console.log(node.classList)
+        // don't include minimap, the controls and the MUI Buttons.
+        if (
```

Comment:
You could return directly a boolean, e.g.
`
 return (!node?.classList?.contains('react-flow__minimap') && !node?.classList?.contains('react-flow__controls') && !node?.classList?.contains('MuiSvgIcon-root') && !node?.classList?.contains('MuiButtonBase-root'))
`